src/MumbleProtocol.cpp

```diff
@@ -335,7 +335,6 @@ namespace Protocol {
 		m_audioMessage.Clear();
 
 		static_assert(AudioContext::BEGIN == 0, "AudioContext::BEGIN is not zero (breaks assumption)");
-		static_assert(AudioContext::END >= 0, "AudioContext::END is negative (breaks assumption)");
```

Comment:
Let's change this assertion to check for `> 0` instead of `>= 0` and adapt the error message accordingly. That should get rid of the compiler warning and still allow us to have this fail-safe check in place (after all `END` should not only be not negative but also not equal to zero, as we assume (for now) that there will be at least one audio context in existence).

Reply:
I didn't notice that, thanks! Made the change and it compiled on my system fine.